schemas/user/auth-user-summary.yml

```diff
@@ -22,20 +22,22 @@ properties:
 
   first_name:
     type: string
+    readOnly: true
     description: |
       The user's first name.
 
   last_name:
     type: string
+    readOnly: true
     description: |
       The user's last name.
 
-required:
-  - first_name
-  - last_name
+  permissions:
+    type: array
+    readOnly: true
+    items:
+      $ref: '#/definitions/permission/permission'
+    description: |
+      The permissions granted to the user.
```

Comment:
At the moment the seed auth api endpoint just returns the list of permissions related to that token, not the user data. I guess we could change it to include that?